### PR TITLE
[PAN-2022] added docker vulnerabilities analysis

### PR DESCRIPTION
**Pull request by juanmanuel-tirado**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x]  Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Added GH job to detect docker vulnerabilities.

## Related Tickets & Documents

NA

## QA Instructions

NA

## [optional] Are there any post deployment tasks we need to perform?

### CODE REVIEW

**Comment by juanmanuel-tirado** [comment]
Some changes had to be done following #7

File: .github/workflows/docker-vulnerabilities.yaml [hunk 1]
```diff
@@ -0,0 +1,74 @@
+name: DockerVulnerabilities
+on:
+  push:
+    branches:
+      - main
+  pull_request: 
+  #schedule:
```
Comment (jacekv) [inline .github/workflows/docker-vulnerabilities.yaml:7]: Want to have the schedule in or not? What's the goal?

**Review by jacekv — COMMENTED** [review]
LG2M

File: .github/workflows/docker-vulnerabilities.yaml (hunk 1 above)
Comment (juanmanuel-tirado) [reply .github/workflows/docker-vulnerabilities.yaml:7]: That's to force at least one run to see it works. In the next commit, I will make it only run when merged into main and on a cron job.

**Review by jacekv — APPROVED** [review]

**Review by jpantos — APPROVED** [review]

**Pull request merged**